src/components/date/weekday/weekday.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+import classicTheme from '../../../style/themes/classic';
+import Weekday from './weekday.component';
+
+describe('Weekday', () => {
+  const shallowRender = (props, renderer = shallow) => {
+    return renderer(<Weekday { ...props }>sample children</Weekday>);
```

Comment:
you could just return `TestRenderer.create(<Weekday { ...props }>sample children</Weekday>)` as there is no other renderer to choose

Reply:
changed, thanks